src/STEMMUS_SCOPE.m

```diff
@@ -345,6 +345,10 @@
         % Calculate the index of the bottom layer level
         [GroundwaterSettings.indxBotmLayer, GroundwaterSettings.indxBotmLayer_R] = groundwater.calculateIndexBottomLayer(GroundwaterSettings.soilThick, GroundwaterSettings.gw_Dep);
         [depToGWT_strt, indxGWLay_strt] = groundwater.findPhreaticSurface(SoilVariables.hh, KT, GroundwaterSettings.soilThick, GroundwaterSettings.indxBotmLayer_R);
+
+        % update soil temperature
+        T = SoilVariables.T;
+        TT = SoilVariables.TT;
```

Comment:
```suggestion
        % update soil temperature
         SoilVariables.T(1:GroundwaterSettings.indxBotmLayer) = GroundwaterSettings.tempBotm;
        T = SoilVariables.T;
        SoilVariables.TT(1:GroundwaterSettings.indxBotmLayer) = GroundwaterSettings.tempBotm;
        TT = SoilVariables.TT;
```

Reply:
this [block](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/9d9588fdc0ce26b4bf3b8aea8933d0fccbd2fb3f/src/%2Benergy/calculateBoundaryConditions.m#L16-L24) can be refactored as: `tempBotm = SoilVariables.TT(indxBotm);`, no ndeed for this[ if block](https://github.com/EcoExtreML/STEMMUS_SCOPE/blob/9d9588fdc0ce26b4bf3b8aea8933d0fccbd2fb3f/src/%2Benergy/calculateBoundaryConditions.m#L21-L23).